### PR TITLE
Chase

### PR DESCRIPTION
**Pull request by johnstoc13**
Added modals. They are ready for JS to make them work. Added some more styling. Updated home page buttons, and rearranged the navbar for home screen prior to login.

### CODE REVIEW

File: public/assets/js/game.js [hunk 1]
```diff
@@ -282,6 +282,7 @@ $(function () {
     // Axios to get the game data
     getCards($gameTeamSelection.val(), $gameQuestionSelection.val());
     // Show game instructions
+    $gameForm.css("display", "none");
```
Comment (hilbug) [inline public/assets/js/game.js]: I intentionally had those dropdowns stay there so they could see the question and team drop-downs while playing the game. So I'm wondering why we're now going to hide them? Also if they wanted to change the question when the instructions come up, hiding that section would not allow them to do that.

File: public/assets/css/style.css [hunk 2]
```diff
@@ -69,10 +69,6 @@ a.nav-link:hover,
     background-color: #5b247a;
 }
 
-/* .learn-more {
```
Comment (hilbug) [inline public/assets/css/style.css:72]: Confirmed with Chase this removal was OK. ~hilary

**Review by hilbug — CHANGES_REQUESTED** [review]
Can you review my comment on public/assets/js/game.js? ~Hilary

File: public/assets/js/game.js (hunk 1 above)
Comment (johnstoc13) [reply public/assets/js/game.js]: Change removed

**Review by hilbug — APPROVED** [review]
No changes to game.js now- OK. ~Hilary

**Pull request merged**